client/src/modules/events/pages/eventPage.tsx

```diff
@@ -130,6 +131,7 @@ export const EventPage: NextPage = () => {
     });
 
     if (ok) {
+      if (!isLoggedIn) await login();
```

Comment:
A nitpick, you can't see the cancel rsvp button and interact with it without logging in, so the check will always be `isLoggedIn`